buildman/manager/executor.py

```diff
@@ -696,7 +696,7 @@ def __init__(self, *args, **kwargs):
         super(KubernetesExecutor, self).__init__(*args, **kwargs)
         self.namespace = self.executor_config.get("BUILDER_NAMESPACE", "builder")
         self.image = self.executor_config.get(
-            "BUILDER_CONTAINER_IMAGE", "quay.io/quay/quay-builder-podman:stable"
+            "BUILDER_CONTAINER_IMAGE", "quay.io/projectquay/quay-builder:3.7.0"
```

Comment:
I'm curious @bcaton85, how will the 3.7.0 tag get built for the quay-builder image? ATM I see we only have the unstable and rc tags https://quay.io/repository/projectquay/quay-builder?tab=tags.

Reply:
I'm not familiar with the build process for the builder, I'm actually looking for that information myself. If we build to latest I'll just switch that tag to that.

Reply:
There's a [github workflow](https://github.com/quay/quay-builder/blob/master/.github/workflows/build-and-publish.yaml) that builds unstable versions of the redhat-* branches on every commit (i.e 3.7-unstable).
In addition to that Quay.io is setup to [build master](https://quay.io/repository/projectquay/quay-builder?tab=builds) on new commits - this is how the `latest` tag is built, so AFAIK you can use that.

Reply:
hmm not sure I follow @HammerMeetNail - doesn't quay/releases trigger https://github.com/quay/quay-builder/blob/master/.github/workflows/build-and-publish.yaml? that's tagging "unstable" images from redhat-* branches, i.e `3.7-unstable`. Or do I misread something?